app/src/main/java/com/google/ytdl/MainActivity.java

```diff
@@ -107,6 +107,8 @@ protected void onCreate(Bundle savedInstanceState) {
         getWindow().requestFeature(Window.FEATURE_INDETERMINATE_PROGRESS);
         super.onCreate(savedInstanceState);
 
+        mUploadsListFragment = new UploadsListFragment(getApplicationContext());
```

Comment:
I needed to add in a new fragment constructor so that a context can be passed  to the uploadListFragment class as you cannot pass in the context of the fragment into GoogleApiClient.